vertical-pod-autoscaler/README.md

```diff
@@ -12,49 +12,113 @@ It can both down-scale pods that are over-requesting resources, and also up-scal
 
 Autoscaling is configured with a
 [Custom Resource Definition object](https://kubernetes.io/docs/concepts/api-extension/custom-resources/)
-called [VerticalPodAutoscaler](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1beta1/types.go).
+called [VerticalPodAutoscaler](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1beta2/types.go).
 It allows to specify which pods should be under vertically autoscaled as well as if/how the
 resource recommendations are applied.
 
 To enable vertical pod autoscaling on your cluster please follow the installation
 procedure described below.
 
-
 # Installation
 
-### Notice on switching from alpha to beta (<0.3.0 to >=0.3.0)
+The current default version is Verical Pod Autoscaler 0.4.0
+
+**NOTE:** version 0.4 requires at least Kubernetes 1.11 to work (needs certain
+Custom Resource Definition capabilities). With older Kubernetes versions we
+suggest using the [latest 0.3 version](https://github.com/kubernetes/autoscaler/blob/vpa-release-0.3/vertical-pod-autoscaler/README.md) 
+
+### Notice on switching to v1beta2 version (0.3.X to >=0.4.0)
 
-Between versions 0.2.x and 0.3.x there is an alpha to beta switch which includes
-a change of VPA apiVersion from `poc.autoscaling.k8s.io/v1alpha1` to `autoscaling.k8s.io/v1beta1`.
-The safest way to switch is to use `vpa-down.sh` script to tear down
-the old installation of VPA first. This will delete your old VPA objects that
-have been defined with `poc.autoscaling.k8s.io/v1alpha1` apiVersion.
-Then use `vpa-up.sh` to bring up the new version of VPA and create you VPA
-objects from the scratch, passing apiVersion `autoscaling.k8s.io/v1beta1`
-(See [example](./examples/hamster.yaml)).
+In 0.4.0 we introduced a new version of the API - `autoscaling.k8s.io/v1beta2`.
```

Comment:
I'd add a word why the change is introduced:
- avoid misuse of selectors (e.g. too wide or too narrow label selector)
- API aligned with HPA

Reply:
Added below.